app/services/XmlBuilderService.scala

```diff
@@ -132,15 +132,36 @@ class XmlBuilderService {
                 buildAndEncodeElem(event.alreadyInNcts,"AlrInNCTCTL29")
               }
             </CTLCTL> {
-               buildIncident(event.eventDetails)(arrivalNotificationRequest)
+               buildIncident(event.eventDetails, event.seals)(arrivalNotificationRequest)
             }
           </ENROUEVETEV>
       }
   }
 
-  private def buildIncident(event: EventDetails)(implicit arrivalNotificationRequest: ArrivalNotificationRequest): NodeSeq = event match {
-    case incident: Incident =>
-      <INCINC> {
+  private def buildSeals(seals: Option[Seq[Seal]]): NodeSeq =
+    seals.fold(NodeSeq.Empty) {
+      seals =>
+        val sealsXml = seals.map {
+          seal =>
+            <SEAIDSI1>
+            <SeaIdeSI11>
+              {seal.numberOrMark}
+            </SeaIdeSI11>
+            <SeaIdeSI11LNG>GB</SeaIdeSI11LNG>
```

Comment:
this should be EN not GB, check with CL012 for language codes, does this need hard coded or would this be DE is applied in Germany, I think its OK but needs to be checked.